css-canvas-medical/html/implementation.html

```diff
@@ -0,0 +1,209 @@
+    <link rel="stylesheet" href="../css/footer.css" />
+    <link rel="stylesheet" href="../css/implementation.css" />
+  </head>
+  <body>
+    <!-- Header Section -->
+    <section class="header-row-container display-flex">
+      <div class="column-one-third">
+        <a href="index.html">
+          <img class="logo" src="../images/logo-blue.svg" alt="canvas" />
+        </a>
+      </div>
+      <nav>
+        <ul class="nav-bar column-two-thirds display-flex">
+          <li class="nav">
+            <a href="product.html" class="hyperlink"> Product</a>
+          </li>
+          <li class="nav">
+            <a href="#" class="hyperlink"> Implementation </a>
+          </li>
+          <li class="nav">
+            <a href="about-us.html" class="hyperlink"> About Us </a>
+          </li>
+          <li class="nav">
+            <a href="blog.html" class="hyperlink"> Blog </a>
+          </li>
+          <li class="nav">
+            <button class="contact-us">
+              <a href="contact-us.html" class="hyperlink-button">
+                Contact Us
+              </a>
+            </button>
+          </li>
+        </ul>
+      </nav>
+    </section>
+    <hr class="nav-bar-hr" />
+
+    <!-- Banner Section -->
+    <section class="banner container">
+      <div class="overlay display-flex">
+        <div class="text-container display-flex">
+          <div class="textbox column-half">
+            <h3 class="title">
+              Canvas will be the second best investment you've ever made,
+              guaranteed.
+            </h3>
+            <p class="text-content">
+              Becoming a physician will endure as your all-time best investment
+              decision, and on behalf of your community, we’re grateful for it.
+              When you choose to invest in Canvas, we'll ensure you list Canvas
+              as the second best investment you’ve ever made for your practice,
+              guaranteed. Contact us when you're ready to discuss the details.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="content-container">
+      <h4 class="content-title">
+        Calculate an estimate of your ROI with Canvas
+      </h4>
+      <div class="content">
+        <h6 class="sub-heading">
+          <span class="numbering">1.</span> Making the Decision To Switch
+        </h6>
+        <p class="content-body">
+          You should only move forward with Canvas if and when you have
+          determined that we far outperform other vendors and offer an unmatched
+          return on investment. Our goal is to give you as much information as
+          possible to make this determination one way or the other. We will
+          provide access to a sandbox of the Canvas application for you to get a
+          feel for the platform as well as references to current customers to
+          talk about their experience.
+        </p>
```

Comment:
I'm not sure what `content-body` is applying, but since you gave it to so many different elements, you might have been able to utilize inhertience here. Give the `<div class="content">` whatever styling you want this `p` to inherit, and then you can just override the styles for the `h6` and the `<span>` if you didn't want them to inherit them as well